.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the Docker image as Default mode 
+      run: |
+        pwd
+        cd ..
+        ls -ltr
```

Comment:
Try also by adding `export DOCKER_BUILDKIT=1` before building

Reply:
Buildkit doesn't seems to be supported by github action. We need to get rid of this mount, last time I checked it was an experimental feature.

Reply:
I found this discussion https://github.com/docker/github-actions/issues/12#issuecomment-600540752 where somebody said that DOCKER_BUILDKIT is supported on Github Actions. I do not know if it is true or not, but maybe we could try to insert `env: DOCKER_BUILDKIT:1 ` as reported in this yml file https://github.com/docker/github-actions/blob/826b584752ffa6abc1a2d94da26f348e3e4f73d0/.github/workflows/main.yml#L26